src/workerd/api/hibernatable-web-socket.h

```diff
@@ -29,8 +29,12 @@ class HibernatableWebSocketEvent final: public ExtendableEvent {
   struct ItemsForRelease {
     jsg::Ref<WebSocket> webSocketRef;
     kj::Own<kj::WebSocket> ownedWebSocket;
+    kj::Array<kj::String> tags;
```

Comment:
Probably worth a comment for each property. Should also update the comment below for `prepareForRelease`.